.idea/modules.xml

```diff
@@ -2,6 +2,7 @@
 <project version="4">
```

Comment:
modules.xml file should not be committed to VCS as it different for everyone based on their OS and environment variables.